test/test-applications/integrations/TestApplication.Smoke/Program.cs

```diff
@@ -47,16 +46,14 @@ public static void Main(string[] args)
         }
     }
 
-    private static void EmitTraces()
+    private static void EmitTracesAndLogs()
```

Comment:
nit: maybe just wise to comment somewhere that the scope of the activity needs to be alive when logging occurs.
Without realizing the connection this method seems like waiting for refactor / separation😄